resources/newssite/news-site-css/src/css/a11y.css

```diff
@@ -41,6 +43,53 @@ but useful for people who use screen readers */
     }
 }
 
+/* High Contrast / Forced-Colors:
+  Apply forced-colors class to parent element
+  to manually envoke force-colors on child elements.
```

Comment:
I'm not sure I understand this comment...

Reply:
removed comment, which was outdated and confusing.